Extension/package.json

```diff
@@ -6118,28 +6118,27 @@
-        "generateOptionsSchema": "node ./tools/prepublish.js && node ./out/tools/generateOptionsSchema.js",
-        "generate-native-strings": "node ./tools/prepublish.js && gulp generate-native-strings",
-        "translations-export": "node ./tools/prepublish.js && gulp generate-native-strings && gulp translations-export",
-        "translations-generate": "node ./tools/prepublish.js && set NODE_OPTIONS=--no-experimental-fetch && gulp translations-generate",
-        "translations-import": "node ./tools/prepublish.js && gulp translations-import",
-        "copy-walkthrough-media": "node ./tools/prepublish.js && gulp copy-walkthrough-media",
-        "prepublishjs": "node ./tools/prepublish.js",
-        "pretest": "tsc -p test.tsconfig.json",
-        "lint": "gulp lint",
-        "unitTests": "tsc -p test.tsconfig.json && node ./out/test/unitTests/runTest.js",
-        "internalUnitTests": "tsc -p test.tsconfig.json && mocha ./out/test/internalUnitTests/test*.js",
-        "integrationTests": "tsc -p test.tsconfig.json && node ./out/test/integrationTests/languageServer/runTest.js",
-        "intelliSenseFeaturesTests": "tsc -p test.tsconfig.json && node ./out/test/integrationTests/IntelliSenseFeatures/runTest.js",
-        "import-edge-strings": "node ./import_edge_strings.js",
-        "postinstall": "npx vscode-dts dev && npx vscode-dts main"
+        "scripts": "ts-node -T .scripts/scripts.ts",
+        "show": "ts-node -T .scripts/clean.ts show",
+        "clean": "ts-node -T .scripts/clean.ts",
+        "test": "ts-node -T .scripts/test.ts",
+        "code": "ts-node -T .scripts/code.ts",
+        "verify": "ts-node -T .scripts/verify.ts",
+        "prep": "yarn copy-walkthrough-media && yarn generate-native-strings && yarn translations-generate",
+        "lint": "eslint -c .eslintrc.js src test ui .scripts",
+        "compile": "(yarn verify prep || yarn prep) && tsc --build tsconfig.json",
+        "watch": "(yarn verify prep || yarn prep )&& tsc --build tsconfig.json --watch",
+        "rebuild": "yarn clean && yarn prep && yarn compile",
+        "vscode:prepublish": "yarn clean && yarn webpack",
+        "webpack": "yarn prep && tsc --build ui.tsconfig.json && webpack --mode production --env vscode_nls",
+        "generate-native-strings": "ts-node -T ./.scripts/generateNativeStrings.ts",
+        "generate-options-schema": "ts-node -T ./.scripts/generateOptionsSchema.ts",
+        "copy-walkthrough-media": "ts-node -T ./.scripts/copyWalkthruMedia.ts",
+        "translations-export": "yarn generate-native-strings && gulp translations-export",
+        "translations-generate": "set NODE_OPTIONS=--no-experimental-fetch && gulp translations-generate",
+        "translations-import": "gulp translations-import",
+        "import-edge-strings": "ts-node -T ./import_edge_strings.ts",
+        "postinstall": "npx vscode-dts dev && npx vscode-dts main && yarn prep"
```

Comment:
Would a "yarn install" script be good to add? Previously, yarn install would be automatically run from the existing npm scripts, so having a script in the list might make the "yarn install" more visible.

Reply:
Can't, and don't need it... `yarn install` is a built-in command, and the `postinstall` command is run automatically at the end of `yarn install` (or `npm install`)

Reply:
Previously, there was a script `"compile": "node ./tools/prepublish.js` and prepublish.js would call yarn.install. Since that is  gone now, adding a replacement script would allow it to appear in the npm scripts UI for users to discover it there.

<img width="240" alt="image" src="https://github.com/microsoft/vscode-cpptools/assets/19859882/cae64c93-b38d-45cd-9483-8115066e3f16">